src/components/UserAvatar/UserAvatar.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import { pipe, toUpper, trim } from 'ramda';
+import PropTypes from 'prop-types';
+
+import { Button } from '../Button';
+import { ButtonColors, ButtonVariants } from '../Button/Button.enums';
+import { Sizes, UserAvatarProps } from './UserAvatar.types';
+import {
+  getFontSize,
+  getFontWeight,
+  getLineHeight,
+  getRadii,
+  pxToRem,
+} from '../../utils/helpers';
+import { UserAvatarSizes } from './UserAvatar.enums';
+
+const UserAvatarSmall = css`
+  padding: ${pxToRem(4, 2.5)};
+`;
+const UserAvatarMedium = css`
+  padding: ${pxToRem(8, 6.5)};
+`;
+
+const userAvatarSizes = {
+  [UserAvatarSizes.sm]: UserAvatarSmall,
+  [UserAvatarSizes.md]: UserAvatarMedium,
+};
+
+const StyledButton = styled(Button)<{ size: Sizes }>`
+  border-radius: ${getRadii('round')};
+  font-size: ${getFontSize('md')};
+  font-weight: ${getFontWeight('regular')};
+  line-height: ${getLineHeight('md')};
+  ${({ size }) => userAvatarSizes[size]};
```

Comment:
We should set width rather than manipulate with padding, we have button heights defined in the theme so we can use those to make it perfect circle.
```suggestion
  width: ${({ size, theme }) =>
    pipe(getButtonHeight(size), pxToRem)({ theme })};
```